app/src/androidTest/java/com/google/jetpackcamera/utils/UiTestUtil.kt

```diff
@@ -192,15 +195,60 @@ fun deleteFilesInDirAfterTimestamp(
     return hasDeletedFile
 }
 
-fun doesImageFileExist(uri: Uri, prefix: String): Boolean {
-    val file = uri.path?.let { File(it) }
-    if (file?.exists() == true) {
-        val mimeType = URLConnection.guessContentTypeFromName(uri.path)
-        return mimeType != null && mimeType.startsWith(prefix)
+fun doesFileExist(uri: Uri): Boolean = uri.path?.let { File(it) }?.exists() == true
+
+fun doesMediaExist(uri: Uri, prefix: String): Boolean {
+    if (prefix == IMAGE_PREFIX) {
+        return doesImageExist(uri)
+    } else if (prefix == VIDEO_PREFIX) {
+        return doesVideoExist(uri, prefix)
     }
     return false
 }
 
+fun doesImageExist(uri: Uri): Boolean {
+    val bitmap = uri.path?.let { path -> BitmapFactory.decodeFile(path) }
+    val mimeType = URLConnection.guessContentTypeFromName(uri.path)
+    return mimeType != null && mimeType.startsWith(IMAGE_PREFIX) && bitmap != null
+}
+
+// this does not work with the image capture tests...
```

Comment:
If this is the case, then you should be validating arguments. Add something like this to the beginning of the function:
```
require(prefix == VIDEO_PREFIX) { "doesVideoExist() only works for videos. Can't handle prefix: $prefix" }
```